conf/layer.conf

```diff
@@ -12,3 +12,12 @@ BBFILE_PRIORITY_raspberrypi = "9"
 # Additional license directories.
```

Comment:
Hi @otavio . This looks good. Just a small NIT: could you use somethine like "layer.conf: Add Dynamic..." as a git log subject?

Reply:
Sure; anything else?